src/options.js

```diff
@@ -602,12 +602,14 @@ export function maybeNamed(things) {
   return isIterable(things) ? named(things) : things;
 }
 
-// TODO Accept other types of clips (paths, urls, x, y, other marks…)?
-// https://github.com/observablehq/plot/issues/181
 export function maybeClip(clip) {
   if (clip === true) clip = "frame";
   else if (clip === false) clip = null;
-  else if (clip != null) clip = keyword(clip, "clip", ["frame", "sphere"]);
+  else if (
+    clip != null &&
+    !["FeatureCollection", "Feature", "Polygon", "GeometryCollection", "MultiPolygon"].includes(clip.type)
```

Comment:
I don’t think we should restrict this to specific GeoJSON types. For example, you could have a FeatureCollection of a single Point feature, so why allow FeatureCollection but not Point? Also, a Sphere geometry should be treated the same as _sphere_.

Reply:
You can't clip to a "Point" but I see your point. I've changed it to a duck-typing test of any GeoJSON object.